client/src/static/mockedDataDemo.ts

```diff
@@ -42,7 +41,7 @@ export const buildInitialModelsList = ({ SIR_PN, SIR_FN, SEIR_PN, SEIRD_PN, SIRD
           },
         },
         {
-          donuType: Donu.Type.GROMET_PRT,
+          donuType: Donu.Type.GROMET_PNC, // to replace with GrFN type once donu puts those in
```

Comment:
I like to add `TODO` in comments that needs to be fixed, so once in a while it's simple to just search for it and easily keep track of them.

Reply:
Thanks, added.